### PR TITLE
Various improvements in the docs

### PR DESCRIPTION
**Pull request by sylvestre**
* fix md warnings
* improve the display of some pages
* add a check in the CI

### CODE REVIEW

File: docs/src/multicall.md [hunk 1]
```diff
@@ -12,6 +13,7 @@ coreutils [util] [util options]
 The `--help` flag will print a list of available utils.
 
 ## Example
-```
+
+```shell
```
[the remark this answers is not in the document]
Comment (sylvestre) [reply docs/src/multicall.md:17]: Shell is probably better. bash is too specific

File: src/uu/dd/dd.md [hunk 2]
```diff
@@ -10,117 +11,116 @@ Copy, and optionally convert, a file system resource
 
 ## After Help
 
-OPERANDS:
```
[the remark this answers is not in the document]
Comment (sylvestre) [reply src/uu/dd/dd.md:13]: unparsed = ? :)

[the remark this answers is not in the document]
Comment (sylvestre) [reply src/uu/dd/dd.md:13]: I think it is good enough, we can probably add md support in terminal ? :)

[the remark this answers is not in the document]
Comment (tertsdiepraam) [reply src/uu/dd/dd.md:13]: I definitely want to. But there wasn't a library for it that I could find. I do agree that it's fine for now.

**Review by tertsdiepraam — APPROVED** [review]

**Pull request merged**